tgui/packages/tgui/interfaces/AirlockElectronics.js

```diff
@@ -0,0 +1,111 @@
+// code\game\machinery\doors\airlock_electronics.dm
+import { useBackend } from '../backend';
+import { Button, Divider, Flex, Section } from '../components';
+import { Window } from '../layouts';
+import { AccessList } from './common/AccessList';
+import { Fragment } from 'inferno';
+
+export const AirlockElectronics = (props, context) => {
+  return (
+    <Window resizable>
+      <UnrestrictedAccess />
+      <Divider />
+      <ChooseAccess />
+    </Window>
+  );
+};
+
+const UnrestrictedAccess = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    unrestricted_dir,
+  } = data;
+  return (
+    <Section title="Access Control">
+      <Flex direction="column">
+        <Flex.Item bold mb={1}>
+          Unrestricted Access Settings
+        </Flex.Item>
+        <Flex.Item>
+          <Button width={5.7}
+            icon="arrow-up"
+            content="North"
+            selected={data.unrestricted_dir === 1 ? "selected" : null}
```

Comment:
```suggestion
            selected={unrestricted_dir === 1 ? "selected" : null}
```
`data.` not required for these since you did the override up here
```js
	  const {
    unrestricted_dir,
  } = data;
```